phoenixRest/views/agenda/instance.py

```diff
@@ -45,4 +49,27 @@ def get_event(context, request):
 def get_applications(context, request):
```

Comment:
Line 44 + 49
Not that it has any effect but these functions should be given a propper name to make the code clearer